pkg/actors/internal/reminders.go

```diff
@@ -33,6 +33,7 @@ type LookupActorFn func(ctx context.Context, actorType string, actorID string) (
 type StateStoreProviderFn func() (string, TransactionalStateStore, error)
 
 // RemindersProvider is the interface for the object that provides reminders services.
+// Deprecated: this will be removed after we have migrated to Scheduler Service.
```

Comment:
There are dependencies on this, should not be deprecated

Reply:
Indeed

Reply:
https://github.com/cicoyle/dapr/pull/71